html/index.php

```diff
@@ -270,7 +274,7 @@ function popUp(URL)
 ?>
 
 <?php
-if($config['enable_footer'] == 1) {
+if($config['enable_footer'] == 1 && (isset($vars['bare']) && $vars['bare'] != "yes") || !isset($vars['bare'])) {
```

Comment:
I think this condition makes the config option obsolete in form of :
`False && False || True`
Its sufficient to drop the last OR.